sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -65,12 +65,14 @@ void ValidateScopesIfNotSanitized()
         [Test]
         [TestCase(AuthMethod.ConnectionString, TestName = "GettingTurnCredentialsWithConnectionString")]
         [TestCase(AuthMethod.KeyCredential, TestName = "GettingTurnCredentialsWithKeyCredential")]
+        [TestCase(AuthMethod.TokenCredential, TestName = "GettingTurnCredentialsWithTokenCredential")]
         public async Task GettingTurnCredentialsGeneratesTurnCredentials(AuthMethod authMethod)
         {
             CommunicationIdentityClient client = authMethod switch
```

Comment:
Now that it is consistent, maybe better to extract this client creation into a method and use it across

Reply:
Done.